### ISSUE TITLE
backslashes added to :root declarations

### ISSUE THREAD

**Issue by kevinSuttle** [postcss/postcss-custom-properties#54]
See https://github.com/pascalduez/postcss-apply/issues/11

<img width="1436" alt="screen shot 2016-09-20 at 5 36 14 pm" src="https://cloud.githubusercontent.com/assets/95672/18689843/3bcc0154-7f59-11e6-9f26-89bb5c910d58.png">

**Comment by MoOx** [comment]
Nothing is modifying custom properties except to remove those. If you want to help, please make a PR with a failing test.

**Comment by kevinSuttle** [comment]
> Nothing is modifying custom properties except to remove those.

Can you clarify?

**Comment by pascalduez** [comment]
@kevinSuttle Be quite with opening so many issues, it's getting hard to follow :)
From your screenshot the slash is applied to custom property sets, is this also happening with "just" custom properties?

As said in pascalduez/postcss-apply/issues/11 I tried to debug it with now success, but the plugins didn't seemed to be applied. Too much players on the game (Webpack2, Storybook, etc.).
We would need to narrow down to a bare minimum use case.

**Comment by MoOx** [comment]
> Can you clarify

Just read the code of this plugin.

The only backslashes in the code are here https://github.com/postcss/postcss-custom-properties/blob/12ad2bf75c40338f629f8b3d47c921ba30a5f98a/index.js#L15
and it's a regex to match, not replace. Anyway, unless you provide a minimal test case (a failing test will be great) I won't be able to help. Also, you are using this module for a while, so if you take a minute, you know it's not the culprit.

**Comment by kevinSuttle** [comment]
Thanks fellas. I know you're spread super thin (especially today). Just trying to provide as much information as possible.
